travis-ci/vmtest/mkrootfs.sh

```diff
@@ -123,20 +115,34 @@ mkdir -m 755 "$root/etc/init.d" "$root/etc/rcS.d"
 cat > "$root/etc/rcS.d/S10-mount" << "EOF"
 #!/bin/sh
 
-/bin/mount -a
+set -eux
+
+/bin/mount proc /proc -t proc
+# Try mounting devtmpfs
+/bin/mount devtmpfs /dev -t devtmpfs || true
+/bin/mount sysfs /sys -t sysfs
+/bin/mount bpffs /sys/fs/bpf -t bpf
+/bin/mount debugfs /sys/kernel/debug -t debugfs
+
+echo 'Listing currently mounted file systems'
+/bin/mount
```

Comment:
debug leftover or you'd like to leave that here permanently?

Reply:
I think it's useful to have mounts listed in CI execution log, since I've run into mounting issues while bumping this version of disk image. Wdyt?

Reply:
yeah, that's fine, especially that now logs are collapsed anyway